### PR TITLE
Feat: Added Login-signup pages

### PR DESCRIPTION
**Pull request by Abhijeet43**
- Added Signup page
- Added Login page
- Added Images and modular css
Login Page Demo: https://novakart.netlify.app/pages/authentication/login.html
Signup Page Demo: https://novakart.netlify.app/pages/authentication/signup.html

### CODE REVIEW

**Review by starver20 — APPROVED** [review]
Code looks good to merge. Nice UI.👍🏻

**Review by mohammed-sas — APPROVED** [review]

**Review by mohammed-sas — APPROVED** [review]

**Review by AtulPant2704 — APPROVED** [review]

File: pages/authentication/auth.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+.section-img {
+  padding: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+form {
```
Comment (Husain01) [inline pages/authentication/auth.css:27]: Just a suggestion: Add a border-radius of a few pixels

**Review by Husain01 — APPROVED** [review]
Looks really good. Nice work

File: pages/authentication/auth.css (hunk 1 above)
Comment (Abhijeet43) [reply pages/authentication/auth.css:27]: ok.. will add

**Review by Husain01 — APPROVED** [review]
Good to go

**Pull request merged**